BridgeCityBicycleCoop/workstand
# Update project to run on Heroku

To help simplify our infrastructure, I have changed the project to be able to deployed and run on Heroku. 

## Changes
1. Simplify project structure
2. Add some dependencies that allow the project to run on Heroku
3. Add feature flag to hide _Bikes_ feature
4. Use Algolia as our secondary index provider 

## Testing
1. Check out (`git checkout heroku`) branch and run project locally (`./manage.py runserver`)
2. Visit https://warm-wildwood-83351.herokuapp.com and login as _admin@workstand.dev_ with password _bike!bike!_ and see if it still works the way we expect!
